Kubernetes/PublicBrowse/dev-publicbrowse-kubernetes-manifest.yml

```diff
@@ -30,6 +30,28 @@ spec:
           limits:
             cpu: 500m
         env:
+        - name: LOGGER_LEVEL
```

Comment:
Think you may need two versions of the file - one for private with login, etc switched off, and one for public where it is switched on.